control-plane/roles/isolated-clusters/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
You will also need to symlink the control plane defaults here as otherwise the variables from there cannot be picked up with the current repos layout.